tests/components/openuv/test_sensors.py

```diff
@@ -0,0 +1,54 @@
+"""Test OpenUV sensors."""
+
+from operator import attrgetter
+
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.components.openuv.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+from homeassistant.helpers.entity_component import async_update_entity
+
+from .conftest import TEST_LATITUDE, TEST_LONGITUDE
+
+
+async def test_sensors(
+    hass: HomeAssistant,
+    set_time_zone,
+    config,
+    config_entry,
+    setup_config_entry,
+    snapshot: SnapshotAssertion,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test all sensors created by the integration."""
+    device_id = f"{TEST_LATITUDE}_{TEST_LONGITUDE}"
+    device = device_registry.async_get_device({(DOMAIN, device_id)})
+
+    assert device is not None
+    assert device == snapshot(name="device")
+
+    entities = entity_registry.entities.get_entries_for_device_id(device.id)
+    assert sorted(entities, key=attrgetter("entity_id")) == snapshot(name="entities")
+
+    for entity in entities:
+        assert hass.states.get(entity.entity_id) == snapshot(name=entity.entity_id)
```

Comment:
use the `snapshot_platform` helper for this

Reply:
Ok, so keep just this one test, rename the file, and use `snapshot_platform`, correct?